contracts/api3-server-v1/proxies/interfaces/IApi3CrossRateReaderProxyV1.sol

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.4;
+
+import "../../../interfaces/IApi3ReaderProxy.sol";
+
+interface IApi3CrossRateReaderProxyV1 is IApi3ReaderProxy {
+    enum CalculationType {
+        Divide1By2,
+        Divide2By1,
```

Comment:
I don't get why both are needed

Reply:
Mainly to support more cross rate combinations. Say you have proxy1 = ETH/USD and proxy2 = SOL/USD. If we wanted a cross rate for ETH/SOL we would divide ETH/USD by SOL/USD but if we wanted SOL/ETH then it would be SOL/USD divided by ETH/USD. An alternative could be to use the reciprocal of the proxies but I wanted to keep it as simple as possible and avoid having to deal with too many if statements based on base and quote checks 🤷🏻

Reply:
I also don't get this. You can change the divisor based on the order of the proxies.

Reply:
https://github.com/api3dao/contracts/pull/472/commits/8e22dff085862c172358f0b5a31def4e6e6ec06d